package.json

```diff
@@ -61,8 +63,14 @@
     "core-js": "^2.6.3",
     "cpx": "^1.5.0",
     "cross-env": "^5.2.0",
+    "cypress": "^3.2.0",
+    "cypress-cucumber-preprocessor": "^1.11.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "1.7.1",
+    "eslint": "^5.3.0",
+    "eslint-config-airbnb-base": "^13.1.0",
```

Comment:
you shouldn't need eslint, eslint-plugin-import or eslint-config-airbnb-base in this package.json as they are already installed via carbon-factory

Reply:
fixed